src/lib/components/templates/Chat.svelte

```diff
@@ -42,17 +101,48 @@
 	<SpokenText --align="center" slot="middle" text="Gesproken bericht" />
 	<Help slot="right" />
 </Header>
-<main>
-	{#each messages as message (message.id)}
-		{#if message.sender.id === userId}
-			<audio controls src={message.file} type="audio/ogg" class="you" />
-		{:else}
-			<audio controls src={message.file} type="audio/ogg" />
+<div>
+	<main>
+		{#each messages as message, index ((message.id, index))}
+			{#if message.file}
+				<AudioPlayer
+					file={message.file}
+					--align={message.sender.id === userId ? 'flex-end' : 'flex-start'}
+				/>
+			{:else}
+				<MessageCloud
+					text={message.content}
+					--align={message.sender.id === userId ? 'flex-end' : 'flex-start'}
+				/>
+			{/if}
+		{/each}
+		{#if isUser}
+			{#if messages.length}
+				{#if !submitted}
+					<form
+						bind:this={form}
+						on:submit|preventDefault={() => {
+							submitted = true
+							resolveLetter(letterId, radioVal)
+							postResolveStatusMessage(letterId, radioVal)
+						}}
+					>
+						<label for="resolve">Ik heb genoeg uitleg gekregen</label>
+						<fieldset>
+							<input type="radio" name="resolve" id="yes" value="Ja" bind:group={radioVal} />
+							<label for="yes">Ja</label>
+							<input type="radio" name="resolve" id="no" value="Nee" bind:group={radioVal} />
+							<label for="no">Nee</label>
+						</fieldset>
+						<input type="submit" value="Verstuur" />
```

Comment:
Instead of having radio inputs with a submit button, create two buttons with different values that submit straight away. This looks a bit cleaner and gets rid of the user having to interact twice with the application for one simple operation.